money-monkey-web/components/CategoryPieChart.tsx

```
@@ -0,0 +1,136 @@
+  className,
+  sums,
+}: {
+  className: string;
+  sums: CategorySum[];
+}) {
+  const { categories } = useContext(UserSettingsContext);
+
+  const total = useMemo(
+    () =>
+      sums.reduce(
+        (x, y) => ({ ...x, total_cents: x.total_cents + y.total_cents }),
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)